tests/unittests/RecommendationSystemTest.cpp

```diff
@@ -567,94 +568,109 @@ class RecommendationSystemTest : public BackendTest {
   }
 
   /// Builds a simple graph, returns back input var and save node through refs.
-  static void createSimpleRecSysGraph(
-      Module &mod_, PlaceholderBindings &bindings_, Function *F_,
-      llvm::ArrayRef<Placeholder *> lengths, Placeholder *denseData,
-      llvm::ArrayRef<size_t> embSizes, size_t embDim, llvm::StringRef funcName,
-      bool quantizeSLWS, bool quantizeFC, bool gatherWeights) {
-    EXPECT_EQ(lengths.size(), embSizes.size());
+  void createSimpleRecSysGraph(Module &mod, PlaceholderBindings &bindings,
+                               Function *F, llvm::ArrayRef<size_t> embSizes,
+                               size_t embDim, llvm::StringRef funcName,
+                               bool quantizeSLWS, bool quantizeFC,
+                               bool gatherWeights) {
+    EXPECT_EQ(tableSizes.size(), embSizes.size());
+
+    // Create the tables.
+    std::vector<Placeholder *> lengths(tableSizes.size());
+    for (unsigned int i = 0; i < lengths.size(); i++) {
+      lengths[i] = mod.createPlaceholder(ElemKind::Int32ITy, {miniBatch},
+                                         "SL" + std::to_string(i), false);
+    }
+
+    auto *denseData = mod.createPlaceholder(ElemKind::FloatTy,
+                                            {miniBatch, denseDim}, "denseData",
+                                            false); // Dense dim can be anything
```

Comment:
"Dense dim" means the param denseDim ? If so, may be just put /* ..*/ in front of "denseDim"